errbot/config-template.py

```diff
@@ -93,6 +93,12 @@
 # For example CORE_PLUGINS = ('ACLs', 'Backup', 'Help') you get those names from the .plug files Name entry.
 # For absolutely no plug: CORE_PLUGINS = ()
 
+# Defines an order in which the plugins are getting their callbacks. Useful if you want to have plugins do
+# pre- or post-processing on messages.
+# The 'None' tuple entry represents all the plugins that aren't to be expicitely ordered. For example, if
```

Comment:
For example. I have a system (separate from the bot) which posts messages on a channel on behalf of other users (it does some sort of redirection from another separate chat system). All of those messages are posted by a "redirector" user, but they all follow the same text structure, with the real username in the message.

I want to have an ErrBot plugin which pre-processes messages from the redirector to replace the "from" header with the username as parsed in the message text, and set the body to the actual content of the message. This needs to happen first, so any other plugin will see that message as coming from the "real" user, rather than from the redirector.

In that case, I need to enforce a "first this, then everything else" order. I would use something like `('RedirectorFixup', None)` as `PLUGINS_CALLBACK_ORDER` to express this relationship.